### PR TITLE
ISAICP-5174: Added user's origin to its profile.

### PR DESCRIPTION
**Pull request by junkuncz**

### CODE REVIEW

File: web/themes/joinup/templates/user/user--group-header.html.twig [hunk 1]
```diff
@@ -50,6 +50,12 @@
             <span class="stats__text">{{ 'Member for'|t }}:</span>
             <span class="stats__text bold">{{ member_for }}</span>
           </div>
+          {% if content.field_user_nationality|render %}
```
Comment (idimopoulos) [inline web/themes/joinup/templates/user/user--group-header.html.twig]: Do we need the render function here? I tried it and it works without the render function. If you remove it from the view mode, it does not show, it does not log any errors. But here we are rendering it once more.
Let's remove it (I know it is like this in the rest) if there is no reason for it.

File: tests/features/user/profile.feature [hunk 2]
```diff
@@ -58,8 +58,12 @@ Feature: User profile
     # A user should not be able to edit the profile page of another user.
     When I go to the public profile of "Domenico Ghirlandaio"
     Then I should not see the link "Edit"
+    # Verify that the user's "Country of origin" field is visible on its profile.
+    When I go to the public profile of "Leonardo Da Vinci"
+    Then I should see the text "Country of origin:" in the "Header" region
+    And I should see the link "Italy" in the "Header" region
```
Comment (idimopoulos) [inline tests/features/user/profile.feature]: In the screenshot attached in the roiginal ticket, this is not a link. Plus, we do not have anything styled for the taxonomy pages. Please, change this into the "featured" view mode of the user to not link to the original entity.

File: web/themes/joinup/templates/user/user--group-header.html.twig (hunk 1 above)
Comment (junkuncz) [reply web/themes/joinup/templates/user/user--group-header.html.twig]: Really!
Changed!

File: tests/features/user/profile.feature (hunk 2 above)
Comment (junkuncz) [reply tests/features/user/profile.feature]: Alright!
I also adjusted the tests.

**Pull request merged**